spec/index.html

```diff
@@ -233,13 +269,37 @@ <h3>Graph Statements</h3>
 
   </section>
 
+  <section id="quoted-triples">
+    <h3>Quoted Triples</h3>
+
+    <p>TriG shares the same syntax for representing <dfn data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triples</dfn>
+      as Turtle, including the <a data-cite="RDF12-TURTLE#annotation-syntax">Annotation Syntax</a>.</p>
+
+    <pre id="ex-annotated-triple"
+         class="example trig"
+         data-transform="updateExample"
+         title="TriG annotated triples">
+    <!--
+      BASE <http://example.org/>
+      PREFIX : <#>
```

Comment:
Is there a specific need to not align base and prefix keywords with the other examples?

```suggestion
      @base <http://example.org/> .
      @prefix : <#> .
```

Reply:
These are also valid TriG declarations, and it's good to have some example usage. Some seem to prefer the SPARQL-style.

Note that Turtle has some examples using `PREFIX` and `BASE`, but TriG had none.